uv/meters/avg_meter.py

```diff
@@ -0,0 +1,16 @@
+class AverageMeter:
+
+  def __init__(self):
+    self.reset()
+
+  def reset(self):
+    self.val = 0
```

Comment:
you've stumbled on a powerful idea... aggregators! We built a whole zoo of these at https://github.com/twitter/algebird.

To do this right, you really want to take this note into account from `AveragedValue`:

https://github.com/twitter/algebird/blob/develop/algebird-core/src/main/scala/com/twitter/algebird/AveragedValue.scala#L29

```
 * The mean calculation uses a numerically stable online algorithm
 * suitable for large numbers of records, similar to Chan et. al.'s
 * [[http://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Parallel_algorithm
 * parallel variance algorithm on Wikipedia]]. As long as your count
 * doesn't overflow a Long, the mean calculation won't overflow.
```

This is the more numerically stable version: https://github.com/twitter/algebird/blob/develop/algebird-core/src/main/scala/com/twitter/algebird/MomentsGroup.scala#L98

Reply:
Keras has a bunch of these too: https://www.tensorflow.org/api_docs/python/tf/keras/metrics/Metric

`Sum`, `Mean` etc if you want some inspiration